docs/en/stack/ml/df-analytics/ml-dfanalytics-evaluate.asciidoc

```diff
@@ -163,3 +164,21 @@ The matrix contains the actual labels on the left side while the predicted
 labels are on the top. The proportion of correct and incorrect predictions is 
 broken down for each class. This enables you to examine how the {classanalysis}
 confused the different classes while it made its predictions.
+
+
+[[ml-dfanalytics-class-aucroc]]
+=== Area under the curve of receiver operating characteristic (AUC ROC)
+
+The receiver operating characteristic (ROC) curve is a plot that represents the 
+performance of the classification process at different predicted probability 
+thresholds. It compares the true positive rate for a specific class against the 
+rate of all the other classes combined ("one versus all" strategy) at the 
+different threshold levels to create the curve.
+
+Let's see an example. You have three classes: `A`, `B`, and `C`, you want to 
+calculate AUC ROC for `A`. In this case, the number of correctly classified `A`s 
+(true positives) are compared to the number of `B`s and `C`s that are 
+misclassified as `A`s (false positives).
+
+From this plot, you can compute the area under the curve (AUC) value, which is a 
+number between 0 and 1. The closer to 1, the better the algorithm performance.
```

Comment:
> The close to 1, the better the algorithm performance.

I got to this point and wasn't sure which "algorithm" this sentence was referring to or what aspect of its performance we were judging.  Can we be more specific  in summing this up?  For example, something like this:

> The higher the AUC, the better the model is at predicting As as As, in this case.